applications/asset_tracker_v2/overlay-low-power.conf

```diff
@@ -0,0 +1,17 @@
+#
```

Comment:
This file needs a docstring to explain why I would want this, because the sample is already ultra-low-power.

> The Asset Tracker v2 is a real-time configurable ultra-low-power application firmware for the nRF9160 System in Package (SiP).

So, is this `low-power` mode actually consuming more power?

Reply:
- Comment added in `overlay-low-power.conf
- README changed to `The Asset Tracker v2 is a real-time configurable ultra-low-power capable application firmware for the nRF9160 System in Package (SiP).`
- Explanation added to the low power overlay description:
```
* :file:`overlay-low-power.conf` - Configuration file to achieve the lowest power consumption, which disables features like LED control and logging that consume extra power.
```